hacking/ticket_stubs/question_not_bug.md

```diff
@@ -2,7 +2,7 @@ Hi!
 
 Thanks very much for your interest in Ansible.  It means a lot to us.
 
-This appears to be a user question, and we'd like to direct these topics to the Ansible Forum.
+This appears to be a user question, and we'd like to direct this topic to the Ansible Forum.
```

Comment:
the plural here is to make it more inclusive and not single out 'this topic' but pointing at general policy